src/TokenizedStrategy.sol

```diff
@@ -812,8 +884,9 @@ contract TokenizedStrategy {
         uint256 maxLoss
     ) private returns (uint256) {
         require(receiver != address(0), "ZERO ADDRESS");
-        require(shares <= maxRedeem(owner), "ERC4626: withdraw more than max");
```

Comment:
is this check no longer required?

Reply:
Was moved to the public functions, so that maxRedeem is used on redeems and maxWithdraw on withdraws to avoid extra rounding and more exact error messages.

https://github.com/yearn/tokenized-strategy/blob/access/src/TokenizedStrategy.sol#L614
https://github.com/yearn/tokenized-strategy/blob/access/src/TokenizedStrategy.sol#L570